lib/usageParams.js

```diff
@@ -53,6 +53,11 @@ export default yargs
     type: 'number',
     default: 1000
   })
+  .option('management-host', {
+    describe: 'Management API host',
+    type: 'string',
+    default: 'api.contentful.com'
+  })
```

Comment:
The SDK will handle the default correctly when the tool is used as a node module so we should be fine for now